client/index.jade

```diff
@@ -0,0 +1,15 @@
+doctype html
+html
+
+  head
+    title Hello, React
+
+  body
+
+    h1.alert.alert-info.this-works webpack dev server (with HMR)
+    #app
+
+    script(src="vendor-bundle.js")
+    script(src="app-bundle.js")
+    script.
+      ReactDOM.render(React.createElement(HelloWorldApp, {name: "Stranger"}), document.getElementById('app'))
```

Comment:
@mattrobs The jade file and use of server.js is used for standalone testing of your JS code without Rails. There's a couple use cases for this:

1. You have non-rails developers that don't want to be slowed down by Rails
2. You want hot reloading of CSS and JS code. However, in the latest https://github.com/shakacode/react-webpack-rails-tutorial/, we've updated it so that it does hot reload JS and CSS.